### ISSUE TITLE
Selecting an element in a facet clears selections made in another facet.

### ISSUE THREAD

**Issue by ianfore** [biocaddie/prototype_issues#184]
1. Search for "autism"
The results are shown
2. select the check box for Download under the Accessibility Facet
The box becomes checked and several boxes under the Repositories facet also become checked
3. Check Bioproject under the Repositories facet
That box remains checked but the selection under Accessibility gets cleared
4. Check Download again - the same selection is returned as at step 2

It seems impossible to filter the results by both of these facets.

**Comment by RuilingLiu** [comment]
Hi @ianfore ,

Right now we don't have accessibility information for each dataset, we only have this information for each repository. 

> 2. select the check box for Download under the Accessibility Facet
>     The box becomes checked and several boxes under the Repositories facet also become checked

Selecting "download" will display all search results from all downloadable repositories. 

>3. Check Bioproject under the Repositories facet
> That box remains checked but the selection under Accessibility gets cleared

For a specific repository, we are unable to filter the search results by accessibility.
